Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -92,7 +93,7 @@ class MSMainViewController: UITableViewController, AppCenterProtocol, CLLocation
 
     // Miscellaneous section.
     self.installId.text = appCenter.installId()
-    self.appSecret.text = appCenter.appSecret()
+    self.appSecret.text = prodAppSecret()
```

Comment:
Similar to #1382 did we want to keep it as appCenter.appSecret() or is prodAppSecret() really necessary?